### DISCUSSION TITLE
Disabling bar creation

### DISCUSSION THREAD

**Discussion by Diroshumen** [karlomikus/bar-assistant#359] [Q&A]
Two questions I haven't found answers to yet if anyone knows.

1 - Is there an option to have a  "super" admin account that always sees/controls all bars?
2- Is there a way to disable the "Create bar" button on the Bars page so anyone with an account can't just go creating new bars?

**Comment by karlomikus** [comment]
Hello,
1. There is currently no admin account. As a server owner you can always access the sqlite database, and do some management via queries.
2. Not at the moment, but it's probably a good feature to include in the future

Thanks!
